python/cudf/cudf/core/df_protocol.py

```diff
@@ -46,6 +48,14 @@ class _Device(enum.IntEnum):
     ROCM = 10
 
 
+class _MaskKind(enum.IntEnum):
+    NON_NULLABLE = (0,)
+    NAN = (1,)
+    SENTINEL = (2,)
+    BITMASK = (3,)
+    BYTEMASK = 4
```

Comment:
Prior to this PR, we used raw ints to represent this information.

Reply:
Is it deliberate that `BYTEMASK` is an int, but the other mask kinds are tuples? (They both turn into ints on construction, but the tuple instantiation looks kind of weird.

Reply:
Fixed to just use ints here.